ngDraggable.js

```diff
@@ -574,23 +574,39 @@ angular.module("ngDraggable", [])
 
                         if (config.horizontalScroll) {
                             // If horizontal scrolling is active.
-                            if (lastMouseEvent.clientX < config.activationDistance) {
+
+                            // lastMouseEvent.clientX is undefined when dealing with a touch device, resulting in
+                            // no scrolling when dragging an item to the bottom of the screen
+                            // Seen on Chrome 47.0.2526.111
+                            var clientX = lastMouseEvent.clientX;
+                            if (angular.isUndefined(lastMouseEvent.clientX))
+                              clientX = lastMouseEvent.touches[0].clientX;
+
+                            if (clientX < config.activationDistance) {
                                 // If the mouse is on the left of the viewport within the activation distance.
                                 scrollX = -config.scrollDistance;
                             }
-                            else if (lastMouseEvent.clientX > viewportWidth - config.activationDistance) {
+                            else if (clientX > viewportWidth - config.activationDistance) {
                                 // If the mouse is on the right of the viewport within the activation distance.
                                 scrollX = config.scrollDistance;
                             }
                         }
 
                         if (config.verticalScroll) {
                             // If vertical scrolling is active.
-                            if (lastMouseEvent.clientY < config.activationDistance) {
+
+                            // lastMouseEvent.clientY is undefined when dealing with a touch device, resulting in
+                            // no scrolling when dragging an item to the bottom of the screen
+                            // Seen on Chrome 47.0.2526.111
+                            var clientY = lastMouseEvent.clientY;
+                            if (angular.isUndefined(lastMouseEvent.clientY))
+                              clientY = lastMouseEvent.touches[0].clientY;
```

Comment:
touches is not defined on event itself, it is defined on originalEvent - use lastMouseEvent.originalEvenet.touches[0]
or rather - inject ngDraggable and use ngDraggable.inputEvent(lastMouseEvent).clientY